applications/shards/game/game.soy

```diff
@@ -0,0 +1,93 @@
+{namespace tictactoe}{template .game}
+{@param username: string | null}
+<div>
+    <a href="/login">
+        <button class="logbutton" onclick="sendLogout()">Logout</button>
+    </a>
+    <h4>{$username}</h4>
+</div>
+<div class="header">
+    <h1>Tic Tac Toe</h1>
+    <h3>Do you want to be "X" or "O"?</h3>
+    <button onclick="setX()" id="x">X</button>
+    <button onclick="setO()" id="o">O</button>
+    <button onclick="newGame()">New Game</button>
+</div>
+</div>
+    <div id="tableplace">
+       /*  <table id="gameBoard">
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+            <tr>
+                <td>1</td>
+                <td>2</td>
+                <td>3</td>
+            </tr>
+        </table> */
+    </div>
+
+    <script type="text/javascript">
+        var playerOne = "";
+
+        function setX(){lb}
+            playerOne = "x";
+        {rb}
+
+        function setO(){lb}
+            playerOne = "o";
+        {rb}
+
+        function newGame(){lb}
+            var tbl = document.getElementById("gameboard");
+            if(tbl){lb}
+                tbl.parentNode.removeChild(tbl);
+            {rb}
+            var body = document.getElementById("tableplace");
+            var tbl = document.createElement("table");
+            tbl.setAttribute("id","gameboard");
+            var tbody = document.createElement("tbody");
+            for(var i = 0; i < 3; i++){lb}
+                var tr = document.createElement("tr");
+                for(var j = 0; j < 3; j++){lb}
+                    var td = document.createElement("td");
```

Comment:
Nit: this can be replaced with the following without any loss of readability:

tr.appendChild(document.createElement("td"));